style.less

```diff
@@ -426,10 +426,19 @@ body {
 
 		.main-navigation {
 			.clearfix();
+
+			.full-container {
+				display: flex;
```

Comment:
This should have align-items: center; applied. Without it, the menu items won't be vertically centered.